Docs/source/building/summit.rst

```diff
@@ -1,3 +1,5 @@
+.. _building-summit:
+
 Building WarpX for Summit (OLCF)
```

Comment:
```suggestion
Building WarpX in Summit (OLCF)
```
I am really not sure about this one! Sorry!

Reply:
Hm, `for` or maybe `on` I would say.